app/models/miq_policy.rb

```diff
@@ -369,6 +370,38 @@ def self.seed
     end
   end
 
+  def self.seed_fixtures
+    fixture_file = File.join(FIXTURE_DIR, "miq_policies.yml")
+    fixtures = File.exist?(fixture_file) ? YAML.load_file(fixture_file) : []
+
+    fixtures.each do |policy|
+      rec = seed_attributes(policy.except(:conditions))
+      seed_conditions(rec, policy[:conditions])
+    end
+  end
+
+  def self.seed_attributes(policy)
+    rec = MiqPolicy.find_by_name(policy[:name])
+    if rec.nil?
+      _log.info("Creating MiqPolicy: [#{policy[:name]}] ")
+      rec = MiqPolicy.create!(policy)
+    else
+      rec.attributes = policy
+      if rec.changed?
+        _log.info("Updating MiqPolicy [#{policy[:name]}] ")
+        rec.save
+      end
+    end
+    rec
+  end
+
+  def self.seed_conditions(rec, conditions)
```

Comment:
For clarity, maybe this should be called `seed_link_to_conditions` or a clearer name (naming is hard)?  We're assuming that conditions are already seeded by this point, the current method name confused me at first.